subprojects/tooling-api/src/integTest/groovy/org/gradle/integtests/tooling/r22/ClientShutdownCrossVersionSpec.groovy

```diff
@@ -45,7 +45,6 @@ class ClientShutdownCrossVersionSpec extends ToolingApiSpecification {
         toolingApi.withConnection {}
         then:
         IllegalStateException e = thrown()
```

Comment:
I understand that it was like that before but this line should read `thrown(IllegalStateException)`. I am not sure if we'd assert the correct exception type with the code as is.